server.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const express = require('express');
+const app = express();
+const cors = require('cors');
```

Comment:
Ryan covered more about CORS today in class. 

- We will need to use the middleware. In addition to loading the module, we will need to add: `app.use(cors()` 👍